src/osd/pg_scrubber.cc

```diff
@@ -624,7 +625,7 @@ void PgScrubber::add_delayed_scheduling()
 
   milliseconds sleep_time{0ms};
   if (m_needs_sleep) {
-    double scrub_sleep = 1000.0 * m_osds->osd->scrub_sleep_time(m_flags.required);
+    double scrub_sleep = 700.0 + 1000.0 * m_osds->osd->scrub_sleep_time(m_flags.required);
```

Comment:
There seems to be a problem with the value of scrub_sleep_time conf value.
Looking into it now.
The +700 hack is needed to pass minimal standalone testing